README.md

```diff
@@ -22,3 +22,51 @@ out their documentation for how to build and install it.
 
 There are two modules `esdt.sdt` for signal detection theory (and decision
 variable correlation) and `esdt.pmf` for psychometric function estimation.
+
+## Fitting a psychometric function
+
+Create a 2AFC psychometric function with logistic sigmoid.
+
+```python
+>>> import numpy as np
+>>> from esdt import pmf
+>>> psychometric = pmf.PsychometricFunction(pmf.logistic, 0.5)
+>>> data = np.array([[0.1, 0.4, 5],
+...                  [0.2, 0.4, 5],
+...                  [0.3, 0.6, 5],
+...                  [0.4, 0.8, 5],
+...                  [0.6, 1.0, 5]])
+>>> params, ll = psychometric.fit(data)
+
+```
+
+Note that the data format has stimulus intensity in the first column, then
+the fraction of correct responses and then the number of responses
+collected.
+
+While we connect data, it is often useful to have a quick and dirty way to
```

Comment:
Did you mean collect?

Reply:
Yes. fixed by commit message `connect -> collect`